.github/actions/on_device_tests/action.yaml

```diff
@@ -42,6 +42,14 @@ runs:
           echo "TEST_TYPE=unit_test" >> $GITHUB_ENV
           echo "USE_SHARDING=1" >> $GITHUB_ENV
         fi
+
+        if [[ "${{ matrix.target_platform }}" == "android"* ]]; then
+          # Test retries for android
+          echo "TEST_ATTEMPTS=3" >> $GITHUB_ENV
```

Comment:
I think we should put TEST_ATTEMPTS configuration into https://github.com/youtube/cobalt/blob/main/.github/config/android-x86.json#L3